packages/svrx/lib/util/helper.js

```diff
@@ -5,13 +5,13 @@ const libPath = require('path');
 const libFs = require('fs');
 const _ = require('lodash');
 const os = require('os');
-
 const CONST = require('../constant');
 
+const { PLUGIN_PREFIX } = CONST;
+const scopeAndNameRegex = /^@([a-zA-Z0-9_-]+)\/(.*)$/;
```

Comment:
There's some bugs:

```js
/^@([a-zA-Z0-9_-]+)\/(.*)$/.test('@FOO/bar')
// true
/^@([a-zA-Z0-9_-]+)\/(.*)$/.test('@foo/')
// true
```

Had better use regex of [scoped-regex](https://www.npmjs.com/package/scoped-regex) instead which's widely used.

For group match: `/@([a-z\d][\w-.]+)\/[a-z\d][\w-.]*$/`

Any ideas?

Reply:
changed `scopeAndNameRegex` to `/^@([a-z\d][\w-.]+)\/([a-z\d][\w-.]*)/`